src/components/JourneyPicker/JourneyPicker.tsx

```diff
@@ -95,44 +96,55 @@ const JourneyPicker = (props: Props) => {
         day,
         fromStationId,
         onSelectDay,
-        onSelectJourney,
+        updateJourneyParams,
         onSelectTimeOfDay,
         stationsById,
         stationsByLine,
         time,
         timeRange,
         toStationId,
         disabled,
-        reverse,
     } = props;
 
     const [timeOfDay, setTimeOfDay] = useState(() =>
         typeof time === "number" ? getTimeOfDayOptionForTime(time) : timeOfDayPickerOptions[0]
     );
 
+    const [selectedNavigationKind, setSelectedNavigationKind] = useState(navigationKindOptions[0]);
+
     const fromStation = fromStationId ? stationsById[fromStationId] : null;
     const toStation = toStationId ? stationsById[toStationId] : null;
 
-    const navigationKind = useMemo(
-        () =>
-            navigationKindOptions.find(
-                (option) => option.id === (reverse ? "arrive-by" : "depart-at")
-            )!,
-        [reverse]
-    );
+    // const navigationKind = useMemo(
```

Comment:
I think it might be possible to continue inferring `navigationKind` from the `JourneyParams` properties passed in here? (We'd have to add `departAfter` as a prop). It could go like:

```ts
const navigationKind = useMemo(() => {
    if (departAfter) {
        return "depart-after";
    }
    return reverse ? "arrive-by" : "depart-at";
}, [reverse, departAfter]);
```

Where possible I find that it's nice to avoid duplicating values that can be computed from upstream props into local state, as they can get out of sync over time if we're not careful.

However, if you're committed to removing this code then please delete the commented-out version before merging :)